game/entity/entity.py

```diff
@@ -0,0 +1,17 @@
+from abc import abstractmethod
+
+
+class Entity:
+    def __init__(self, name: str):
+        self._type: str = name
+
+    @abstractmethod
+    def draw(self, screen, font_size) -> None:
+        pass
+
+    def get_type(self) -> str:
```

Comment:
maybe make the type an enum? not str

Reply:
and make 
@property
def type(self)...

Reply:
ok thank you, I completely forgot about property decorator.